applications/kubernetes/audit_logging/rule.yml

```diff
@@ -0,0 +1,56 @@
+documentation_complete: true
+
+prodtype: eks
+
+title: Enable audit Logs
+
+description: |-
+  The audit logs are part of the EKS managed Kubernetes control plane logs that
+  are managed by Amazon EKS. Amazon EKS is integrated with AWS CloudTrail, a
+  service that provides a record of actions taken by a user, role, or an AWS
+  service in Amazon EKS. CloudTrail captures all API calls for Amazon EKS as
+  events. The calls captured include calls from the Amazon EKS console and code
+  calls to the Amazon EKS API operations.
+
+rationale: |-
+  Exporting logs and metrics to a dedicated, persistent datastore such as
+  CloudTrail ensures availability of audit data following a cluster security
+  event, and provides a central location for analysis of log and metric data
+  collated from multiple sources.
+
+severity: 'unknown'
+
+identifiers:
+  cce@eks: CCE-89333-9
+
+references:
+  cis@eks: 2.1.1
+
+ocil_clause: 'audit logging is not enabled'
+
+ocil: |-
+  Perform the following to determine if CloudTrail is enabled for all regions:
+  Via the Management Console
+  <ol>
+    <li>Sign in to the AWS Management Console and open the EKS console at https://console.aws.amazon.com/eks</li>
+    <li>Click on Cluster Name of the cluster you are auditing</li>
+    <li>Click Logging</li>
+    <li>Ensure all 5 choices are set to Enabled</li>
+  </ol>
+  If any of the choices are disabled, this is a finding.
+
+  Via the CLI
+  <tt>aws --region "<i>${REGION_CODE}</i>" eks describe-cluster --name "<i>${CLUSTER_NAME}</i>" --query 'cluster.logging.clusterLogging[?enabled==true].types'</tt>
+  If no results are returned, this is a finding.
+
+fix: |-
```

Comment:
For my own education, how is this attribute translated into the DataStream? I know the OCIL is then available as `question_text` element, but how do we handle (esp. through CO) the `fix` attribute?

Reply:
That's a good question. I don't think we do (at least I haven't been able to find a way to get that to show up in rules created by the compliance-operator).

I started writing these following some of the advice in the CaC guides and `fix` seems like the most appropriate place for the remediation.

Reply:
Well - I think we can back out this rule from this PR since it landed in a separate patch [0] where I kept the remediation in the OCIL.

I'd be curious to know if someone from the CaC project could advise on using `fix` in this context. I think we'd need to add it to the datastream before we can update the CO to consume it.

[0] https://github.com/ComplianceAsCode/content/commit/07b62a3cd819c1c5d33c16153ba24a5eb79a163d

Reply:
AFAIK the contents of the "fix" key doesn't propagate to the built SCAP source data stream. It's used only by utils/create_srg_export.py. Recently, @Mab879 started to use the "fix"  for RHEL9 STIG, for example in https://github.com/ComplianceAsCode/content/pull/8321. @Mab879 do you have some more information about this "fix"?

Reply:
@jan-cerny That is correct. To the best of the knowledge only `utils/create_srg_export.py` uses the `fix` key. The `fix` was designed to be human readable.